README.md

```diff
@@ -1,25 +1,11 @@
-🚨🚨🚨
 
-**This library is no longer actively maintained.** The Authy API has been replaced with the [Twilio Verify API](https://www.twilio.com/docs/verify). Twilio will support the Authy API through November 1, 2022 for SMS/Voice. After this date, we’ll start to deprecate the service for SMS/Voice. Any requests sent to the API after May 1, 2023, will automatically receive an error.  Push and TOTP will continue to be supported through July 2023.
+# Twilio Verify Devise
 
-[Learn more about migrating from Authy to Verify.](https://www.twilio.com/blog/migrate-authy-to-verify)
+This is a [Devise](https://github.com/heartcombo/devise) extension to add [Two-Factor Authentication with Twilio Verify](https://www.twilio.com/docs/verify) to your Rails application.
 
-Please visit the Twilio Docs for:
 * [Verify + Ruby (Rails) quickstart](https://www.twilio.com/docs/verify/quickstarts/ruby-rails)
 * [Twilio Ruby helper library](https://www.twilio.com/docs/libraries/ruby)
 * [Verify API reference](https://www.twilio.com/docs/verify/api)
```

Comment:
I'd remove these as you don't need to know them to understand how to use the software.